src/CodeFormatter/Program.cs

```diff
@@ -28,12 +29,44 @@ internal static class Program
 
         private static int Main(string[] args)
         {            
-            return Parser.Default.ParseArguments<ListOptions, FormatOptions>(args)
+            return Parser.Default.ParseArguments<
+                ListOptions, 
+                ExportOptions,
+                FormatOptions>(args)
               .Return(
                 (ListOptions listOptions) => RunListCommand(listOptions),
+                (ExportOptions exportOptions) => RunExportOptionsCommand(exportOptions),
                 (FormatOptions formatOptions) => RunFormatCommand(formatOptions),
                 errs => 1);
+        }
+
+        private static int RunExportOptionsCommand(ExportOptions exportOptions)
+        {
+            int result = 1;
+
+            try
+            {
+                PropertyBag allOptions = new PropertyBag();
+ 
+                // The export command could be updated in the future to accept an arbitrary set
+                // of analyzers for which to build an options XML file suitable for configuring them.
+                foreach (IOptionsProvider provider in FormattingEngine.GetOptionsProviders(s_defaultCompositionAssemblies))
+                {
+                    foreach (IOption option in provider.GetOptions())
+                    {
+                        allOptions.SetProperty(option, option.DefaultValue);
+                    }
+                }
+                allOptions.SaveTo(exportOptions.OutputPath, id: "codeformatter-options");
+                Console.WriteLine("Options file saved to:" + Path.GetFullPath(exportOptions.OutputPath));
+                result = 0;
+            }
+            catch (IOException) { }
```

Comment:
I have removed this exception handling. errors now flow to entry point as is true with other verbs.